### DISCUSSION TITLE
Stacked /sethomes

### DISCUSSION THREAD

**Discussion by Agaloth** [EssentialsX/Essentials#4389] [Ideas and Feature Suggestions] [answered]
Basically the current /sethome system works with permissions, my idea would be to get a permission like essentials.sethome.stacked.1 (That permission would add 1 more sethome to the current rank if that rank already has a permission like essentials.sethome.multiple.default

So let's say the default rank (essentials.sethome.multiple.default) has 5 sethomes then adding the stacked permission to the player (essentials.sethome.stacked.1) would give him access to do 6 sethomes.

This feature is mostly important for servers that want to make rankups for sethomes but have people with purchasable ranks that ends up not getting more homes since they have a higher permission.

**Comment by AlessioGr** [comment]
Completely agree, that feature would be pretty useful for if you have multiple rank tracks which all modify the number of homes you can have.

**Comment by JRoy** [comment, marked as answer]
This doesn't work because people will expect you to be able to add `essentials.sethome.stacked.1` to each inherited level when that isn't possible. Just create a group for each rank that has the amount of homes you want them to have.
